daprdocs/content/en/reference/components-reference/supported-name-resolution/nr-kubernetes.md

```diff
@@ -0,0 +1,23 @@
+---
+type: docs
+title: "Kubernetes DNS Name Resolution Provider"
```

Comment:
This is a "component" not a "provider". Anywhere where you have the term provider, should be replaced with component